### DISCUSSION TITLE
Check and change cut order

### DISCUSSION THREAD

**Discussion by vincentBenet** [arkypita/LaserGRBL#2345] [Ideas]
I am doing laser cut on leather, but sometimes the laser cut the outer bounds first and then cutting small holes in the leather. As the form is cutted it is moving because of the blowing of the fan. This often the job.

My features request to solve that is:
    - Visualisation of cut order of shapes
    - Possibility to change / choose the cut order heuristic during file loading widget

**Comment by Tribo-2dot0** [comment]
se você montar seu design em Inkscape pode ordenar a sequencia de corte dando ordem às camadas e objetos... camadas inferiores serão cortadas primeiro, o mesmo vale para os objetos dentro da camada.
